src/services/user.service.js

```diff
@@ -0,0 +1,45 @@
+import { User } from '../models/user.js';
+import { v4 as uuidv4 } from 'uuid';
+import { emailService } from './emai.services.js';
+import { ApiError } from '../exceptions/api.error.js';
+
+const getByEmail = (email) => {
+  return User.findOne({ where: { email } });
+};
+
+const registerUser = async (name, email, password) => {
+  const activationToken = uuidv4();
+  const isUserExist = await getByEmail(email);
+
+  if (isUserExist) {
+    throw ApiError.badRequest('User with this email already exist.');
```

Comment:
This is an error related to the task requirements. The method `badRequest` is being called on `ApiError`, but it seems like `badRequest` is not a static method of `ApiError`. Ensure that `badRequest` is correctly defined and accessible.